qiskit_addon_sqd/fermion.py

```diff
@@ -193,20 +192,20 @@ def optimize_orbitals(
     """
     if isinstance(bitstring_matrix, tuple):
         warnings.warn(
-            "Passing a length-2 tuple of sorted addresses to define the subspace is deprecated. Users "
+            "Passing a length-2 tuple of base-10 determinants to define the subspace is deprecated. Users "
```

Comment:
I think simply "determinants" is more accurate than "base-10 determinants." They are simply integers, without reference to a "base."

Reply:
Updated the warning message